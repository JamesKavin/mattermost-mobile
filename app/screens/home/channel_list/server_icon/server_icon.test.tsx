// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {renderWithIntlAndTheme} from '@test/intl-test-helper';

import Icon from './server_icon';

test('Server Icon Component should match snapshot', () => {
    const {toJSON} = renderWithIntlAndTheme(
        <Icon/>,
    );

    expect(toJSON()).toMatchSnapshot();
});
