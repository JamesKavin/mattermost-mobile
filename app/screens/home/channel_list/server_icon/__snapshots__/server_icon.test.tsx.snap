// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server Icon Component should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 0,
      "height": 40,
      "justifyContent": "center",
      "left": 16,
      "overflow": "hidden",
      "position": "absolute",
      "top": 10,
      "width": 40,
      "zIndex": 10,
    }
  }
>
  <Icon
    color="rgba(255,255,255,0.56)"
    name="server-variant"
    size={24}
  />
</View>
`;
